src/ted2zim/entrypoint.py

```diff
@@ -192,7 +195,7 @@ def main():
         logger.error(f"FAILED. An error occurred: {exc}")
         if args.debug:
             logger.exception(exc)
-        raise SystemExit(1)
+        raise SystemExit(1) from None
```

Comment:
This is raised by ruff [B904](https://docs.astral.sh/ruff/rules/raise-without-from-inside-except/)
Since we log the "inner" exception just before, I see no point to raise it further (or it would be displayed twice)

Reply:
So it is 😉